### PR TITLE
test: fix and improve ci

### PR DESCRIPTION
**Pull request by jost-s**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -5,7 +5,7 @@
     versions.url = "github:holochain/holochain?dir=versions/0_2";
     holonix.url = "github:holochain/holochain";
     holonix.inputs.versions.follows = "versions";
-    holonix.inputs.holochain.url = "github:holochain/holochain/holochain-0.2.2";
+    holonix.inputs.holochain.url = "github:holochain/holochain/holochain-0.2.3-beta-rc.1";
```
Comment (steveej) [inline flake.nix:8]: we don't build this holochain tag on CI. see [here](https://github.com/holochain/holochain/blob/8521cee803aadb6e51fa967d57006eadf7861630/versions/0_2/flake.nix#L5) what's used for the 0_2 versions flake

Comment (jost-s) [reply flake.nix]: Right, thanks for clarifying @steveej

Comment (ThetaSinner) [reply flake.nix]: The simple solution with that one is that we should get 0.2.3 released. Just waiting on https://github.com/holochain/holochain/pull/2951 and we can go ahead with that

Comment (jost-s) [reply flake.nix]: 👍

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -64,5 +64,8 @@ jobs:
             target
           key: ${{ runner.os }}-build-${{ hashFiles('Cargo.lock') }}
 
+      # - name: tmate session
+      #   uses: mxschmitt/action-tmate@v3
```
Comment (mattyg) [inline .github/workflows/test.yml:68]: remove?

**Review by mattyg — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (jost-s) [reply .github/workflows/test.yml:68]: Okay

**Pull request merged**